parts/language/lang_vi.lua

```diff
@@ -254,12 +254,12 @@ return {
         "Đây chỉ là một trò chơi xếp gạch *thông thường*. Không, thật đấy, chỉ có vậy thôi",
         "Lấy cảm hứng từ C2/IO/JS/WWC/KOS etc.",
         "",
-        "Được xây dựng bằng LÖVE",
+        "Được tạo bằng LÖVE",
```

Comment:
(P/s: this part is a bit of messy for me so I spoke in Vietnamese, sorry)
For short, this string I, Shard and some others have discussed at least... 3 times

--------------------------------------------------------------------------------

Anh Sạc, cái này không biết có nên phải **nhắc lại** nữa hay không
(Dù sao em cũng mong đây là lần cuối cùng phải bàn tới cái chuyện này, em nói thật ;-;)

Cái chỗ này em có thêm cái lựa chọn là "Chạy bằng engine LOVE", nhưng đây có khi là **lần thứ 3** bàn tới chuyện này rồi nên nếu mà anh hoang mang thì thử hỏi lại bên cộng đồng TVN thêm một lần nữa xem